layouts/partials/announcement.html

```diff
@@ -1,6 +1,6 @@
 {{ if .Page.Param "announcement" }}
 <section lang="en" id="announcement" style="background-color:{{ .Page.Param "announcement_bg" }}">
-  <main>
+  <main-announce>
```

Comment:
This is a bit odd @kbhawkey 

`<main-announce>` isn't part of HTML
How about `<aside class="main-announcement">`?

Reply:
Right, as it stands there are two main elements.
What are the other options?

Reply:
From an implementation perspective you can _probably_  drop `<main>` altogether, though that has consequences – site-wide the CSS selector `main .main-section` sets the max width for full-page sections.

If you want, I can take a look at this/take over it and deal with the consequences. the `<main>` tag's use across this site is kind of messy anyways. Some cleanup might be beneficial.

Reply:
I was thinking of `<aside>` rather than `<main>`; how does that sound?

Reply:
```diff
diff --git a/layouts/partials/announcement.html b/layouts/partials/announcement.html
index cea1eb402..8f2710350 100644
--- a/layouts/partials/announcement.html
+++ b/layouts/partials/announcement.html
@@ -1,6 +1,6 @@
 {{ if .Page.Param "announcement" }}
 <section lang="en" id="announcement" style="background-color:{{ .Page.Param "announcement_bg" }}">
-  <main>
+  <aside>
     <div class="content announcement main-section">
 
       <h4 class="announcement">
@@ -9,6 +9,6 @@
       <p class="announcement">{{ T "announcement_message" | markdownify }}</p>
 
     </div>
-  </main>
+  </aside>
 </section>
-{{ end }}
\ No newline at end of file
+{{ end }}
```

Reply:
Geez, this has become very exciting.
How about temporarily fixing with an `aside` so that the anchors show and then
create an issue to work on fixing `announcement.html`.

Reply:
@kbhawkey that sounds good (please assign the issue to me). Forewarning: I suspect far more changes will be needed due to that CSS selector.

Reply:
Note: `frontpage-announcement.html` has a `main` element.